registry/configuring_registry_storage/configuring-registry-storage-nutanix.adoc

```diff
@@ -0,0 +1,32 @@
+:_content-type: ASSEMBLY
+[id="configuring-registry-storage-nutanix"]
+= Configuring the registry for Nutanix
+include::_attributes/common-attributes.adoc[]
+:context: configuring-registry-storage-nutanix
+
```

Comment:
Is there any text we can add here so that we don't have one heading immediately following another?

See the "Dividing information" section of [Headings](https://www.ibm.com/docs/en/ibm-style?topic=format-headings#dividing-information) in IBM Style.